TO-DK Changes PKIO (#20)
Logius-standaarden/Digikoppeling-Beveiligingsstandaarden-en-voorschriften: ch03_PKIoverheid certificaten.md

| --- | --- | --- |

> | PKI001 | Gebruik OIN in subject serial number veld is verplicht | Dit is afgesproken met de TSPs in de Digikoppeling Overeenkomsten. Zij verstrekken PKIoverheid certificaten met het OIN in het subject.serialnumber field conform de OIN systematiek als het een certificaat betreft dat voor Digikoppeling wordt gebruikt. [[PKI-CA]] |

Reviewer: ```suggestion
| PKI001 | Gebruik OIN in `subject:serialNumber` veld is verplicht | Dit is afgesproken met de TSP's in de Digikoppeling Overeenkomsten. Zij verstrekken PKIoverheid certificaten met het OIN in het `subject:serialNumber` field conform de OIN systematiek als het een certificaat betreft dat voor Digikoppeling wordt gebruikt. [[PKI-CA]] |
```
We schrijven doorgaans certificaat velden als een `code` inline en met een colon als markering tussen hoofdveld en subvelden. De meervoudsvorm van TSP schrijven we op z'n Nederlands, met een apostrof.

| PKI002 | PKIoverheid certificaat moet geldig zijn (het mag niet zijn verlopen of ingetrokken). | |
| PKI003<br>(WT004)  | De geldigheid van het certificaat wordt getoetst met betrekking tot de geldigheidsdatum en de Certificate Revocation List(CRL) die voldoet aan de eisen van PKIoverheid. | |
| PKI004<br>(WT005) | De betreffende CRL dient zowel voor de versturende als ontvangende partij te benaderen zijn. | |

> | PKI005 | Het certificaat moet zijn van het type PKIoverheid private root (PKI Staat der Nederlanden Private Root) <br>Voor Serviceaanbieders  en Servicegebruikers geldt dat zij vanaf 1-1-2021 gebruik moeten maken van private root certificaten| PKIoverheid geeft aan dat voor machine-naar-machine (M2M) verkeer (zoals Digikoppeling) Private root certificaten gebruikt dienen te worden.|

Reviewer: Deze tekst doelt op een bepaald type G1 Private certificaat. Deze is beschikbaar in twee varianten: met domeinnamen en zonder domeinnamen. Bij de G4 maken we deze varianten expliciet en benoemen we ze als afzonderlijke certificaattypes: 

De opvolger van "G1 Private Services Server" (zonder domeinnamen) (OID: 2.16.528.1.1003.1.2.8.6) is "G4 Private Other Generic Legal Persons Organization Validated Authentication" (OID: 2.16.528.1.1003.1.2.44.16.25.8)

De opvolger van "G1 Private Services Server" (met domeinnamen) (OID: 2.16.528.1.1003.1.2.8.6) is "G4 Private TLS Generic Devices Organization Validated Server" (OID: 2.16.528.1.1003.1.2.44.15.35.11)

Dit zal terug moeten komen in het document.

Reviewer 2: Bedankt. Dit zetten wij voort in #22.

> ### PKIoverheid programma van eisen

Reviewer: ```suggestion
### PKIoverheid Programme of Requirements
```

1. Een PKIoverheid certificaat dient conform de eisen van het PKIoverheid PvE te worden uitgegeven door de Trust Service Providers (TSP).

1. De te gebruiken certificaten in de productie omgeving voldoen aan de eisen van PKIoverheid (PvE) en de inhoud van de identificerende velden in het certificaat dienen te voldoen aan de afspraken zoals gesteld in de functionele eisen in het document [[[?DK-IDAuth]]]. Met het toepassen van PKIoverheid-certificaten die Digikoppeling compliant zijn, wordt hieraan voldaan.

> 1. Certificaten voor productie wijken af van certificaten voor test doordat zij op verschillende ‘roots’ zijn gebaseerd, respectievelijk 'Staat der Nederlanden Root Private Root’ en ‘Trial PKIoverheid root’.

Reviewer: ```suggestion
1. Certificaten voor productie wijken af van certificaten voor test doordat zij onder een andere 'root' zijn uitgegeven, respectievelijk 'Staat der Nederlanden' (productie) en 'TRIAL PKIoverheid' (test). Deze zijn te vinden op <https://cert.pkioverheid.nl>. 
```
We hanteren doorgaans de term "uitgegeven onder". TRIAL wordt ook geschreven in hoofdletters. Ook stel ik voor een link naar naar de website van PKIoverheid toe te voegen om een brug te slaan naar de specifieke Roots die beschikbaar zijn.

### Geldigheid

> De geldigheid van het certificaat wordt getoetst met betrekking tot de geldigheidsdatum en de Certificate Revocation List(CRL) die voldoet aan de eisen van PKIoverheid. Zie eis PKI002 en PKI003

Reviewer: ```suggestion
De geldigheid van het certificaat wordt getoetst met betrekking tot de geldigheidsdatum en de Certificate Revocation List (CRL) die voldoet aan de eisen van PKIoverheid. Zie eis PKI002 en PKI003. 
```
Spatie.
